### PR TITLE
Bump Mermaid version to 10.0.0

### PR DESCRIPTION
**Pull request by shtratos**
Mermaid version 10.0.0 is the latest, but it has some breaking API changes.

I had to change the `mermaid.render` call as it no longer accepts a callback.
see https://github.com/mermaid-js/mermaid/blob/master/CHANGELOG.md#1000

**N.B. I tested the rendering and it works for correct graph, but it does not render parsing errors.**
I'm very new to node.js, so not sure if my changes are correct. 
Any pointers on how to fix this would be appreciated.

Also:
- I've bumped the version of `css-loader` to resolve the major version conflict with webpack.
- I've added `npm-run-all` as a dev dependency to make `vscode:prepublish` task run on Windows correctly. (Powershell does not recognize `;` as a command delimiter )

### CODE REVIEW

File: package.json
Comment (mjbvz) [inline package.json:98]: Please only include necessary changes in PRs to make them easier to review. The run-all change looks ok but belongs in a separate PR

**Review by mjbvz — CHANGES_REQUESTED** [review]
Yes we do need the parse errors so I can't merge this as is. Maybe someone else can help out with this?

Also can you please not include the package-lock change? I'll update this myself once the PR is merged

**Comment by shtratos** [comment]
I've been seeing this issue in the logs, let's see if the fix helps
https://github.com/mermaid-js/mermaid/issues/4140

File: markdownPreview/mermaid.ts
Comment (sidharthv96) [inline markdownPreview/mermaid.ts:20]: You'll have to add an await here.

**Comment by shtratos** [comment]
Closing in favour of #184

**Pull request closed without merging**
